Base.rte/Scenes/Objects/Bunkers/BunkerSystems/Teleporters/TeleporterA.lua

```diff
@@ -1,74 +1,95 @@
 function Create(self)
-    --Make sure the teleporter list exists.
```

Comment:
I feel like these teleporters should share one script and use preset name or something to track things.
That said, they're gonna be replaced by something more robust at some point anyway so don't worry about it.

Reply:
Yeah the way it's set up is pretty shite. I was again just bettering a lame script.

Reply:
Yeah, and better it you certainly did. As I said, don't worry about it for now.